news/_posts/2020-05-11-cms-security-summit.md

```diff
@@ -0,0 +1,95 @@
+
+
+# Breakout Sessions
+
+The rest of the time was used for breakout sessions, allowing participants to share ideas, experiences around many differents themes defined before the summit.
+
+## Rapid Detection and prevention
+
+Content Management Systems are used by the majority of websites in the world and the number of attacks are still increased every years.
+This has been identified as a priority for everyone to be able to quickly detect attacks and act as fast as possible accordingly.
+
+On the PrestaShop side, the open source project decided to greatly improve the communication around security issues and get closer to (YesWeHack)[https://yeswehack.com/] for the implementation of a Bug Bounty program.
+By doing so, we will detect security issues easier, faster and provide a security fix as fast as possible regarding the criticiy of the issue.
+
+
+## Automatic updates
+
+As everyone already know, the principal risk for a software is its age, keeping it updated is mandatory and considerably reduce the risk of having security issues.
+Even if you think at a T time there are no problems, nobody and nothing can predict the future, a 0 day on PHP, Apache, Nginx ...
+By updating the software you're sure it's compatible with latest system softwares versions.
+
+However, updating a CMS is more risky when you have custom theme, uncompatible modules, modules not following good practices or if [BC Breaks](bc-breaks) are introduced due to security reasons or performance gain.
+
+We're not going to lie to each other, we've clearly putting the autoupgrade module on the back, and that hurts us.
+Everyone know how important and necessary it is for the community.
+
+We spent a lot of time during the summit talking about the self upgrade process, sure we will took time this year improving this module!
+
+## Disclose + Distribute + Query security bulletins in a standardized way?
+
+We paid the price at the beginning of the year, during weeks we sent and re sent emails to [cve.mitre.org](https://cve.mitre.org/) hoping they will update the [CVE-2017-9841](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-9841) about the PHPUnit issue in order to add the CGI/FastCGI context. We are still waiting...
+
+PSR-9 and PSR-10 are always on the side, this is the reason why the open source project decided to use and follow [GitHub Advisories](https://help.github.com/en/github/managing-security-vulnerabilities/about-github-security-advisories), and after weeks of usage, people look happy to see we are more transparent about security issues.
+
+But keep in mind we will never disclose how to reproduce security issues, you will never find any scripts in advisories or in CVE declared by PrestaShop, for your own goods.
+
+Of course it's not perfect, and we will never be. But we learn from the past, and we will do our best to listen the community and improve how we work.
+
+## Release window coordination
```

Comment:
Vague title. A title should summarize the content. To understand this title I had to read the content.

```suggestion
## PrestaShop release schedule will follow PHP ecosystem release schedule
```